buildpack.toml

```diff
@@ -0,0 +1,43 @@
+api = "0.8"
+
+[buildpack]
+  id = "paketo-buildpacks/dotnet-core-aspnet-runtime"
+  homepage = "https://github.com/paketo-buildpacks/dotnet-core-aspnet-runtime"
+  keywords = ["dotnet", "ASP.NET", "runtime"]
+  name = "Paketo Buildpack for .NET Core ASP.NET Runtime"
+  sbom-formats = ["application/vnd.cyclonedx+json", "application/spdx+json", "application/vnd.syft+json"]
+
+  [[buildpack.licenses]]
+    type = "Apache-2.0"
+    uri = "https://github.com/paketo-buildpacks/dotnet-core-aspnet-runtime/blob/main/LICENSE"
+
+[metadata]
+  include-files = ["bin/run","bin/build","bin/detect","buildpack.toml"]
+  pre-package = "./scripts/build.sh"
+
+  [[metadata.dependencies]]
+    cpe = "cpe:2.3:a:microsoft:asp.net_core:6.0:*:*:*:*:*:*:*"
+    deprecation_date = "2024-11-08T00:00:00Z"
+    id = "dotnet-core-aspnet-runtime"
+    name = ".NET Core ASP.NET Runtime"
+    licenses = ["MIT", "MIT-0"]
+    purl = "pkg:generic/dotnet-aspnetcore@6.0.4?checksum=ca3dc696af0a9fc5c7ce052eba38ecf723cbc30d1dc29d8f85c201eff534d73b&download_url=https://download.visualstudio.microsoft.com/download/pr/de3f6658-5d5b-4986-aeb1-7efdf5818437/7df572051df15117a0f52be1b79e1823/aspnetcore-runtime-6.0.4-linux-x64.tar.gz"
+    sha256 = "ca3dc696af0a9fc5c7ce052eba38ecf723cbc30d1dc29d8f85c201eff534d73b"
+    source = "https://download.visualstudio.microsoft.com/download/pr/de3f6658-5d5b-4986-aeb1-7efdf5818437/7df572051df15117a0f52be1b79e1823/aspnetcore-runtime-6.0.4-linux-x64.tar.gz"
+    source_sha256 = "ca3dc696af0a9fc5c7ce052eba38ecf723cbc30d1dc29d8f85c201eff534d73b"
+    stacks = ["io.buildpacks.stacks.bionic"]
+    uri = "https://download.visualstudio.microsoft.com/download/pr/de3f6658-5d5b-4986-aeb1-7efdf5818437/7df572051df15117a0f52be1b79e1823/aspnetcore-runtime-6.0.4-linux-x64.tar.gz"
+    version = "6.0.4"
+
+  [[metadata.dependency-constraints]]
+    constraint = "3.1.*"
+    id = "dotnet-aspnetcore"
```

Comment:
I believe the `id` will need to match the dependency ID `"dotnet-core-aspnet-runtime"`

Reply:
Yep missed that one should be fixed now.